src/elements/types/ExportSpecifier.js

```diff
@@ -0,0 +1,43 @@
+import ModuleSpecifier from '../ModuleSpecifier';
+
+// tests are in ExportNamedDeclaration
+
+export default class ExportSpecifier extends ModuleSpecifier {
+    constructor(childNodes) {
+        super('ExportSpecifier', childNodes);
+    }
+
+    _acceptChildren(children) {
+        console.log(children._elements.map(function(e) {
+            return e._type;
+        }));
+
+        let local;
+        console.log(children._currentElement);
+        console.log(children.isNode('Identifier'));
+        console.log(children.isToken('Identifier'));
```

Comment:
I must be doing something very wrong?

For some reason with `export {x};` `x` is an `Identifier` so `children.passNode('Identifier')` works.
However with  `export {x as y};` `x` is an `Identifier` with  `children.isNode('Identifier')` is false?